src/main/webapp/app/controllers/submission/adminSubmissionViewController.js

```diff
@@ -332,9 +332,13 @@ vireo.controller("AdminSubmissionViewController", function ($anchorScroll, $cont
             return $scope.submission.actionLogs[index].entry;
         };
 
-        // TODO: determine how this is set
         var isUmiRelease = function() {
-            return 'no';
+            var umiReleaseFilterFieldValue = $scope.submission.fieldValues.filter(function (fv) {
```

Comment:
sorry, one more change. umiRelease should by default be 'no' just in case the predicate umi_publication doesnt exist